studio/components/interfaces/BillingV2/Usage/Usage.constants.tsx

```diff
@@ -134,33 +136,80 @@ export const USAGE_CATEGORIES: CategoryMeta[] = [
         chartPrefix: 'Average ',
         unit: 'bytes',
         description:
-          'Billing is based on the average daily database size in GB throughout the billing period.',
+          'Database size refers to the monthly average storage usage, as reported by Postgres. Paid plans use auto-scaling disks.\nBilling is based on the average daily database size used in GB throughout the billing period. Billing is independent of the provisioned disk size.',
```

Comment:
"Billing is independent of the provisioned disk size"

not sure what that means